src/cloudrun-graphql/GraphQLStarter.ts

```diff
@@ -0,0 +1,127 @@
+    tb.copyAsset('docker-compose/docker-compose.ci.yml')
+    tb.copyAsset('docker-compose/docker-compose.local.yml')
+    tb.copyAsset('docker-compose/docker-compose.override.yml')
+    tb.symlink(
+      `${tb.destination}/docker-compose/docker-compose.override.yml`,
+      `${tb.destination}/docker-compose/docker-compose.local.yml`
+    )
+    tb.copyAsset('docker-compose/docker-compose.yml')
+
+    tb.npm.iDev('typescript')
+    tb.npm.iDev('@types/node')
+    tb.npm.i('source-map-support')
+    tb.copyAsset('tsconfig.json')
+    tb.packageJson.addNpmScript(
+      'build:graphql-types',
+      'graphql-codegen --config codegen.yml'
+    )
+    tb.packageJson.addNpmScript(
+      'build:copy-schema',
+      'mkdir -p ./dist/app/api/graphql/schema && cp -r ./src/app/api/graphql/schema ./dist/app/api/graphql/schema'
+    )
+    tb.packageJson.addNpmScript(
+      'build',
+      'npm run build:graphql-types && npm run build:copy-schema && tsc'
+    )
+    tb.packageJson.addNpmScript(
+      'start',
+      'node -r source-map-support/register dist/index.js'
+    )
+
+    tb.npm.i('configuru')
+    tb.npm.i('pino')
+    tb.npm.iDev('pino-pretty')
+    tb.copyAsset('.env.jsonc')
+    tb.mkdir(tb.stringToPath(`${tb.destination}/src`))
+    tb.mkdir(tb.stringToPath(`${tb.destination}/src/app`))
+    tb.mkdir(tb.stringToPath(`${tb.destination}/src/app/api`))
+    tb.mkdir(tb.stringToPath(`${tb.destination}/src/app/api/graphql`))
+    tb.mkdir(tb.stringToPath(`${tb.destination}/src/app/api/graphql/resolvers`))
+    tb.mkdir(tb.stringToPath(`${tb.destination}/src/app/api/graphql/schema`))
+    tb.copyAsset('src/config.ts')
+    tb.copyAsset('src/logger.ts')
+    tb.copyAsset('src/index.ts')
+    tb.copyAsset('codegen.yml')
+
+    tb.copyAsset('src/app/server.ts')
+    tb.copyAsset('src/app/controller.ts')
+    tb.copyAsset('src/app/api/graphql/schema.ts')
+    tb.copyAsset('src/app/api/graphql/resolvers.ts')
+    tb.copyAsset('src/app/api/graphql/resolvers/greeting.resolver.ts')
+    tb.copyAsset('src/app/api/graphql/schema/schema.graphql')
+
+    tb.npm.iDev('mocha')
+    tb.npm.iDev('ts-mocha')
+    tb.npm.iDev('mocha-junit-reporter')
+    tb.npm.iDev('@types/mocha')
+    tb.copyAsset('.mocharc.json', tb.destination)
+    tb.packageJson.addNpmScript('test', 'ts-mocha')
+    tb.packageJson.addNpmScript(
+      'ci-test',
+      'npm run test -- --parallel=false -R mocha-junit-reporter -O=mochaFile=./output/text.xml'
+    )
+    tb.mkdir(tb.stringToPath(`${tb.destination}/src/test`))
+    tb.copyAsset('src/test/helloWorld.test.ts')
+
+    tb.npm.iDev('@ackee/styleguide-backend-config')
+    tb.npm.iDev('prettier')
+    tb.npm.iDev('eslint')
+    tb.copyAsset('.eslint.tsconfig.json')
+    tb.copyAsset('.eslintrc.js')
+    tb.copyAsset('prettier.config.js')
+
+    tb.packageJson.addNpmScript(
+      'prettier',
+      "prettier --check --write '**/*.{ts,js,json,md}'"
+    )
+    tb.packageJson.addNpmScript('lint', "eslint '**/*.ts' -f codeframe --fix")
```

Comment:
This is not liniting graphql files. Please use:

`eslint --ext .ts --ext .graphql src -f codeframe --fix`

Reply:
Where exactly did you put that update? 😸
I don't see any update, when I generate the app, lint is still not lining graphql files.